metals/src/main/scala/scala/meta/internal/worksheets/WorksheetProvider.scala

```diff
@@ -138,6 +141,48 @@ class WorksheetProvider(
     publisher.hover(path, position)
   }
 
+  /**
+   * Check to see for a given path if there is an evaluated worksheet that
+   * matches thie exact input.
+   *
+   * @param path to the input used to search previous  evaluations.
```

Comment:
```suggestion
   * @param path to the input used to search previous evaluations.
```

Reply:
My awesome mac keyboard is starting to leave double spaces all over the place... Fixed!